license.md

```diff
@@ -1,9 +1,9 @@
-Copyright (C) 2007-2022 Laurent Gomila - laurent@sfml-dev.org
+Copyright (C) 2007-2023 Laurent Gomila - laurent@sfml-dev.org
 
 This software is provided 'as-is', without any express or implied warranty. In no event will the authors be held liable for any damages arising from the use of this software.
 
 Permission is granted to anyone to use this software for any purpose, including commercial applications, and to alter it and redistribute it freely, subject to the following restrictions:
 
   1. The origin of this software must not be misrepresented; you must not claim that you wrote the original software.  If you use this software in a product, an acknowledgment in the product documentation would be appreciated but is not required.
   2. Altered source versions must be plainly marked as such, and must not be misrepresented as being the original software.
-  3. This notice may not be removed or altered from any source distribution.
+  3. This notice may not be removed or altered from any source distribution.
```

Comment:
What changed here?

Reply:
Oh it's just a missing trailing newline that somehow got deleted. We can easily add that back and then this PR will be perfect.

Reply:
I'm just copy current `license.md` and paste in my fork and change year. Maybe it's appear because previous commit I'm accidentally format `license.md` in vscode